### DISCUSSION TITLE
Tailscale version for tailscale+mosh

### DISCUSSION THREAD

**Discussion by carlos-solis** [blinksh/blink#1976] [Q&A]
Hi,
I read the documentation about ssh server in Tailscale, and it seems that the version available in the app store does not support the ssh server. In your documentation, you link to the installation page from Tailscale for the app store version.

Could you confirm which version from Tailscale must be installed?

**Comment by iflowerml** [comment]
Hi, thanks for reaching out!

As stated in the [Tailscale SSH Server documentation](https://tailscale.com/kb/1193/tailscale-ssh#prerequisites), the Tailscale SSH’s server component is only available on Linux and MacOS binaries. The iOS version of the app will connect the device to your Tailnet, so then Blink can access over SSH if the Server has that functionality enabled. Tailscale SSH Server is available for the Personal, Premium, and Enterprise plans, to set it up you would have to follow the [instructions on this page](https://tailscale.com/kb/1193/tailscale-ssh).

For purposes of using Blink with Tailscale, [we recommend using mosh](https://docs.blink.sh/integrations/tailscale+mosh#step-1-install-tailscale-on-your-local-network).  Using mosh instead of ssh is strongly encouraged as you have more stable and faster connections.
Please let us know if you have any further questions.
